src/sagas/index.js

```diff
@@ -509,22 +556,17 @@ function* navigateToVmDetails ({ payload: { vmId } }) {
 export function* rootSaga () {
   yield all([
     ...sagasLogin,
-    ...sagasRefresh,
+    ...yield sagasRefresh(),
 
-    takeLatest(CHECK_TOKEN_EXPIRED, doCheckTokenExpired),
+    takeEvery(CHECK_TOKEN_EXPIRED, doCheckTokenExpired),
     takeEvery(DELAYED_REMOVE_ACTIVE_REQUEST, delayedRemoveActiveRequest),
+    takeEvery(NAVIGATE_TO_VM_DETAILS, navigateToVmDetails),
 
     throttle(100, GET_BY_PAGE, fetchByPage),
-    throttle(100, GET_VMS, fetchVms),
-    throttle(100, GET_POOLS, fetchPools),
-
-    takeLatest(GET_ALL_EVENTS, fetchAllEvents),
```

Comment:
do we have a convention for ordering root sagas? if not then let's keep previous ordering

Reply:
Convention is whatever makes sense.